### ISSUE TITLE
images are getting uploaded to cloudinary in a .cache folder with gatsby-remark-images-anywhere

### ISSUE THREAD

**Issue by tonymamo** [cloudinary-devs/gatsby-transformer-cloudinary#103]
Hey,

I'm using gatsby-transformer-cloudinary to utilize `createRemoteImageNode()` in conjunction with frontmatter data saved from NetlifyCMS in markdown files, so I can use gatsby-images. I'm only using createRemoteImageNode on the images in the frontmatter portion of my markdown files, and not doing anything with the images in the body of the markdown files because I couldn't figure out a way to do that with gatsby-transformer-cloudinary.

I was using `gatsby-remark-relative-images` for those body images, sourcing them from a relative folder, but have since removed all image files from my repo since they are in cloudinary now. To get the gatsby-image benefits that gatsby-remark-relative-images provides, I found gatsby-remark-images-anywhere which creates the nodes needed for a gatsby-image.

However, once I added gatsby-remark-images-anywhere, I noticed a .cache folder inside of my `uploadFolder` that I have set in the options for gatsby-transformer-cloudinary. 

I'm only uploading certain images in gatsby-node using createRemoteImageNode, so I'm wondering why it is also uploading other files automatically. I've tried changing the order of the plugins but doesn't seem to matter. Any insights into why this might be happening?

The directory structure in Cloudinary looks like:
```
getpackup/ /* the one set up as my uploadFolder in gatsby-transformer-cloudinary */
|--.cache/
|----caches/
|------gatsby-remark-images-anywhere/
|--------bunch of folders with uid's/
|----------image file
```

Minimal gatsby-config and gatsby-node files below. Thanks!

**gatsby-config:**
```
require('dotenv').config({
  path: `.env.${process.env.NODE_ENV}`,
});

module.exports = {
  flags: {
    FAST_DEV: false,
    PARALLEL_SOURCING: true,
  },
  siteMetadata: {
    ...
  },
  plugins: [
    ...
    {
      resolve: 'gatsby-source-filesystem',
      options: {
        path: `${__dirname}/src/pages`,
        name: 'pages',
      },
    },
    {
      resolve: 'gatsby-source-filesystem',
      options: {
        name: 'images',
        path: `${__dirname}/src/images`,
      },
    },
    {
      resolve: 'gatsby-source-filesystem',
      options: {
        name: 'blog',
        path: `${__dirname}/src/pages/blog`,
      },
    },
    {
      resolve: 'gatsby-transformer-cloudinary',
      options: {
        cloudName: process.env.GATSBY_CLOUDINARY_CLOUD_NAME,
        apiKey: process.env.GATSBY_CLOUDINARY_API_KEY,
        apiSecret: process.env.GATSBY_CLOUDINARY_API_SECRET,
        uploadFolder: 'getpackup',
        fluidMaxWidth: 2048,
        overwrite: false,
      },
    },
    ...
    'gatsby-transformer-sharp',
    'gatsby-plugin-sharp',
    {
      resolve: 'gatsby-transformer-remark',
      options: {
        plugins: [
          {
            resolve: 'gatsby-remark-images-anywhere',
            options: {
              staticDir: 'static',
              sharpMethod: 'fluid',
            },
          },
        ],
      },
    },
    {
      resolve: 'gatsby-plugin-netlify-cms',
      options: {
        publicPath: 'cms',
        modulePath: `${__dirname}/src/cms/cms.ts`,
      },
    }
  ],
};
```

**gatsby-node**
```
const path = require('path');
const { createFilePath } = require('gatsby-source-filesystem');
const { createRemoteImageNode } = require('gatsby-transformer-cloudinary');

...

exports.onCreateNode = async ({
  node,
  actions,
  getNode,
  createNodeId,
  createContentDigest,
  reporter,
}) => {
  const { createNodeField, createNode } = actions;

  const POST_NODE_TYPE = 'MarkdownRemark';

  if (node.internal.type === POST_NODE_TYPE) {
    const value = createFilePath({ node, getNode });
    createNodeField({
      name: 'slug',
      node,
      value,
    });

    if (node.frontmatter && node.frontmatter.heroImage) {
      await createRemoteImageNode({
        url: node.frontmatter.heroImage,
        parentNode: node,
        relationshipName: 'heroImage',
        createNode,
        createNodeId,
        createContentDigest,
        reporter,
      });
    }
};

```

**Comment by SignetOHara** [comment]
Just to chime in, I too would like to know if there's a way to prevent the plugin from automatically uploading images in the project's .cache directory automatically. My situation is I'm using gatsby-source-instagram, which store's the images it fetches in .cache, which is then being uploaded to cloudinary by this plugin, although I'm also not specifying the .cache folder in the options gatsby-config file.

Any help will be much appreciated!

**Comment by Chuloo** [comment]
Currently on this, related to #99

**Comment by SignetOHara** [comment]
Hi Chuloo,

Has there been any luck on this one?

**Comment by raae** [comment]
It seems the plugin does not check the node type, only the node mime/type, when deciding what to upload. I decided to label this a bug as I believe the original intention was to only upload image file nodes created by the gatsby-source-filesystem.

**Comment by cloudinary-raae** [comment]
Released: v2.3.0
